Tasks/AppCenterDistributeV2/Tests/package.json

```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks",
   "devDependencies": {
-    "nock": "9.0.2"
+    "nock": "^13.2.9"
```

Comment:
I checked release notes and since all tests are passing it should be fine - only thing - I'd rather use fixed version so we avoid unnecessary problems in the future.